config.json

```diff
@@ -852,27 +852,47 @@
     {
       "slug": "binary",
       "uuid": "48ca1682-ca1b-4a2c-be55-624620f336a7",
+      "core": false,
```

Comment:
that's unfortunate.

Reply:
Yes, so sorry I missed this in the previous round of fixes.